Test line expect(true).to.be.true fails.

index-test.js line:
    expect(true).to.be.true
yields message:
    TypeError: Cannot read property 'be' of undefined    


Full listing:
// ♥ learn                                                                                                                                                                        
                                                                                                                                                                                  
> java-script-handlebars-templates@0.1.0 test /home/jpfingst-32149/code/labs/javascript-handlebars-templates-v-000                                                                
> mocha --timeout 5000 -R mocha-multi --reporter-options nyan=-,json=.results.json                                                                                                
                                                                                                                                                                                  
 0   __,------,                                                                                                                                                                   
 1   __|  /\_/\                                                                                                                                                                   
 0   _~|_( x .x)                                                                                                                                                                  
     _ ""  ""                                                                                                                                                                     
                                                                                                                                                                                  
  0 passing (2s)                                                                                                                                                                  
  1 failing                                                                                                                                                                       
                                                                                                                                                                                  
  1) index runs:                                                                                                                                                                  
     TypeError: Cannot read property 'be' of undefined                                                                                                                            
     at Context.it (test/index-test.js:16:20)                                                                                                                                    
                                                                                                                                                                                  
                                                                                                                                                                                  
                                                                                                                                                                                  
npm ERR! Test failed.  See above for more details.                                                                                                                                
[23:51:48] (master) javascript-handlebars-templates-v-000                                                                                                                         
// ♥

--- a/test/index-test.js
+++ b/test/index-test.js
@@ -6,13 +6,13 @@ const path = require('path')
 
 
 describe('index', () => {
-  
+
   jsdom({
     src: fs.readFileSync(path.resolve(__dirname, '..', 'index.js'), 'utf-8')
   })
-  
+
 
   it('runs', () => {
-    expect(true).to.be.true
+    expect(true).toBe.true
   })
 })

